# 位置消息
uint8 friendcarid
int16 friendx
int16 friendy
uint8 enemycarid
int16 enemyx
int16 enemyy